TMT Mac/Views/MoviePosterView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  MoviePosterView.swift
+//  TheatricalMovieTrailers
+//
+//  Created by Christoph Parstorfer on 15.10.20.
+//  Ported to macOS by Christoph Parstorfer on 24.12.21.
+//
+
+import SwiftUI
+
+struct MoviePosterView: View {
+    private let filmPosterAspectRatio = CGFloat(0.7063020214)
+    
+    @State private var reflectionDistance: CGFloat
```

Comment:
I feel like this doesn't have to be a `@State` and could just be a `var`, because it doesn't change over the course of the lifetime of the view, but I might be incorrect.

Reply:
You are right!